src/main/io/osd.c

```diff
@@ -628,6 +657,11 @@ void osdDrawElements(void)
         osdDrawSingleElement(OSD_HOME_DIR);
     }
 #endif // GPS
+#ifdef MAG
+    if (sensors(SENSOR_MAG) || sensors(SENSOR_GPSMAG)) {
```

Comment:
This should not be conditional on MAG. `attitude.values.yaw` is calculated with or without a MAG sensor, and will be pretty accurate for some mixers if a GPS is used (check this with the GPS in your car), so there is use for it (and it will even be accurate) under some circumstances without MAG. (Also, note that `SENSOR_GPSMAG` is not set anywhere, so it should not be checked).